.github/workflows/pypi_build_and_images.yaml

```diff
@@ -0,0 +1,39 @@
+# This workflow will build and upload WMCore core services to the production PYPY
+# based on tag releases
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+  # for manual builds on GH Action page please uncomment the following section
+#   workflow_dispatch:
+#     inputs:
+#       name:
+#         description: 'WMCore services'
+
+jobs:
+  build_and_publish_services:
+    name: Build_and_upload_to_pypi
+    strategy:
+      matrix:
+        target: [wmagent, wmagent-devtools, wmcore, reqmon, reqmgr2, global-workqueue, acdcserver, reqmgr2ms-unmerged,
+                 reqmgr2ms-output, reqmgr2ms-rulecleaner, reqmgr2ms-transferor, reqmgr2ms-monitor]
+    uses: ./.github/workflows/pypy_build_template.yaml
+    with:
+      wmcore_component: ${{ matrix.target }}
+    secrets:
+      pypy_token: ${{ secrets.PYPY_PRODUCTION }}
+
+  build_images:
+    name: Build_images
+    needs: [build_and_publish_services]
+    strategy:
+      matrix:
+        target: [wmagent, reqmon, reqmgr2, reqmgr2ms-unmerged, global-workqueue,
+                 reqmgr2ms-output, reqmgr2ms-rulecleaner, reqmgr2ms-transferor, reqmgr2ms-monitor]
+    uses: ./.github/workflows/pypi_images_template.yaml
```

Comment:
how about renaming this file to "docker_images_template.yaml"?

Reply:
done